pkg/constants/constants.go

```diff
@@ -184,5 +187,5 @@ var (
 	TorchTuneEntrypoint = []string{"tune", "run"}
 
 	// TorchTuneImmutableConfigs is the set of immutable configs for the TorchTune Trainer.
-	TorchTuneImmutableConfigs = sets.New(TorchTuneModelOutputDir, TorchTuneTokenizerPath, TorchTuneCheckpointDir)
+	TorchTuneImmutableConfigs = sets.New(TorchTuneModelOutputDir, TorchTuneTokenizerPath, TorchTuneCheckpointDir, TorchTuneTokenizerMergeFile)
```

Comment:
Hi @Electronic-Waste I did that, this takes up the merge config as well from the CTR. I checked the `kubectl describe <node pod>` as well. The `merges_file` config is coming up. But PVC still has no `.txt` file.

Reply:
Now its working, i had to use my [model-initializer](https://hub.docker.com/repository/docker/akashjaiswal03/model-initializer)and [dataset-initializer](https://hub.docker.com/repository/docker/akashjaiswal03/dataset-initializer) image instead of upstream one.

Thanks a lot @Electronic-Waste